setup/azurecmdfiles.wxi

```diff
@@ -3005,6 +3005,186 @@
                             <File Id="fil1EAB5B4A37A81C1E7363673E0D1A6066" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.ServiceBus\System.Net.Http.Primitives.dll" />
                         </Component>
                     </Directory>
+                    <Directory Id="dir01DDB38B2F9C4F474B26B595504B31A1" Name="AzureRM.ServiceFabric">
+                        <Component Id="cmpB4A171450862A582B7D56BDD52F2AF95" Guid="*">
```

Comment:
You should not have .dll-Help.xml files for other projects in your output.  Please clean your repo, rebuild, and regen the installer.